examples/00-pyunits/basic_usage.py

```diff
@@ -269,3 +269,20 @@
 
 meters_per_second.value  # >>> 3.4137599999999995
 meters_per_second.units  # >>> "m s^-1"
+
+
+###############################################################################
+# Using preferred units
+# ~~~~~~~~~~~~~~~~~~~~~
+# Specify a list of units that quantities will automatically convert to.
+
+Quantity.preferred_units(units=["J"])
+
```

Comment:
Something I did not realise initially is that to override a prior selection of a preferred unit, the remove step is manual. Perhaps we should document that.

Reply:
Would it be better to override the unit automatically? @seanpearsonuk

Reply:
@seanpearsonuk I merged this but I guess this comment maybe still needs some discussion.

Reply:
@dnwillia-work @dnwillia @ansaboutin the main thing is, whatever the behaviour is that it's documented.